source/Calamari/Scripts/Octopus.Features.IISWebSite_BeforePostDeploy.ps1

```diff
@@ -467,12 +467,15 @@ if ($deployAsWebSite)
 		# in the deployment process
 		if ($_.certificateVariable) {
 			$sslCertificateThumbprint = $OctopusParameters[$_.certificateVariable + ".Thumbprint"]
-		} else {
+		} elseif ($_.thumbprint){
 			# Otherwise, the certificate thumbprint was supplied directly in the binding
 			$sslCertificateThumbprint = $_.thumbprint.Trim()
+		} else {
+			[Console]::Error.WriteLine("Cannot configure a https binding when neither certificate thumbprint nor certificate variable is not supplied on the binding.")
+			exit 1
 		}
 
-		Write-Host "Finding SSL certificate with thumbprint $sslCertificateThumbprint"
+		Write-Host "Finding SSL certificate with thumbprint $sslC\ertificateThumbprint"
```

Comment:
Is the `\` PowerShell magic? `sslC\ertificateThumbprint`.

Reply:
PEBKAC issue. Fixed.